src/main/java/games/strategy/triplea/ui/AbstractStatPanel.java

```diff
@@ -51,21 +50,21 @@ public AbstractStatPanel(final GameData data) {
 
   public List<PlayerID> getPlayers() {
     final List<PlayerID> players = new ArrayList<>(gameData.getPlayerList().getPlayers());
-    Collections.sort(players, new PlayerOrderComparator(gameData));
+    players.sort(new PlayerOrderComparator(gameData));
     return players;
   }
 
   class ResourceStat extends AbstractStat {
     final Resource resource;
 
-    public ResourceStat(final Resource resource) {
+    ResourceStat(final Resource resource) {
       super();
       this.resource = resource;
     }
 
     @Override
     public String getName() {
-      return resource.getName();
+      return resource == null ? "" : resource.getName();
```

Comment:
Your title implies a change like `Strings.nullToEmpty(resource.getName())`, but your change avoids a NullPointerException. Is that intended?
If yes, feel free to merge

Reply:
Yeah, the title is really not the best. I'll update the commit comment. The issue is that `resource.getName()` would NPE if `resource` is null.